index.html

```diff
@@ -61,6 +61,22 @@ <h3>LET’S CONNECT</h3>
           </li>
         </ul>
       </section>
+      <div class="mobile-menu" id="mymobilemenu">
+        <img
+          src="images/Enabled.png"
+          class="mobile-menu-toggler"
+          alt="mobile-menu-toggler"
+        />
+        <ul class="mobile-menu-items">
+          <li><a href="">Portfolio</a></li>
+          <li><a href="">About</a></li>
+          <li><a href="">Contact</a></li>
+        </ul>
```

Comment:
Great design team🤝

- [ ] As stated in the project requirements, kindly ensure that when the user clicks (or taps) any of the mobile menu options of `Portfolio`, `About`, and `Contact`, the correct part of the page is displayed.